### PR TITLE
feature/eng-3786-add-state-field-to-rounting-legacy

### PR DESCRIPTION
**Pull request by nilsbeck**
# Description

Adds missing field `state`. Also makes use of new `All()` function.

## Changes

* `output.go`
* `main.go`

### CODE REVIEW

File: templates/routing-legacy/main.go [hunk 1]
```diff
@@ -74,15 +74,13 @@ func solver(
 		return FleetOutput{}, err
 	}
 
-	runSolutions := run.Last
```
Comment (merschformann) [inline templates/routing-legacy/main.go]: I am not completely following why we change the default output type of the routing legacy app. The issue doesn't have context either. Can you catch me up? :thinking:

**Review by merschformann — APPROVED** [review]
Just a question about the change in default, but maybe that was the point and I just don't have the info. :relaxed:

File: templates/routing-legacy/main.go (hunk 1 above)
Comment (nilsbeck) [reply templates/routing-legacy/main.go]: I think, I thought for a moment, that we returned all solutions by default in cloud. but that is incorrect. so I reverted this change.

Comment (merschformann) [reply templates/routing-legacy/main.go]: Oh yeah, makes sense. We did return all solutions for the frontend to visualize them. But in normal use, we did not. :relaxed:

**Pull request merged**